rasa_core/policies/ensemble.py

```diff
@@ -55,6 +57,25 @@ def _training_events_from_trackers(training_trackers):
 
         return events_metadata
 
+    def _check_priorities(self) -> None:
+        """Checks for duplicate policy priorities within PolicyEnsemble."""
+
+        priority_dict = {}
+        for p in self.policies:
+            if p.priority in priority_dict:
```

Comment:
remove all the if / else (L65-L68) and replace with
`priority_dict[p.priority].append(type(p).__name__)`
(the defaultdict will automatically create a list if it is not already there, so we do not need to check that)

Reply:
ah dope, I should have assumed when i ran into that problem that python already had it figured out :)

Reply:
no worries, that's what a code review is for - to spread the word :wink: